Automattic/jetpack
# My Jetpack: Add dynamic product prices

This PR introduces fetching the prices for the products we are highlighting in the My Jetpack page from the WPCOM `products` API endpoint.

#### Changes proposed in this Pull Request:

* Fetch product prices dynamically from WPCOM

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to My Jetpack
* Check the `myJetpackInitialState` global var
* Verify that items have a `currency_code` and `full_price` attributes
